writings/updated/02-chapter-1.adoc

```diff
@@ -75,7 +75,7 @@ I have therefore learned Him, or really know Him, or am grown up in Him to a per
 Such possessors of notional faith deceive their own soul, and also deceive others,
 by declaring and boasting of what God has not wrought in them,
 and leading men out beyond the measure of God`'s present work in them.
-But the living faith works powerfully in the Spirit (by love thereto,^
+But the living faith works powerfully in the Spirit (by love thereto^
```

Comment:
This seems janked up, but it's hard to see in the asciidoc because of the footnote.  See the image below.  I might have messed this up in intake.  I'm assuming this is not how you wanted it right?  We could also nix the other comma, or fix it some other way.  Or, if you like to party, we can leave it.

![2018-10-31_13-33-21](https://user-images.githubusercontent.com/7050938/47807584-d9eae580-dd12-11e8-82f3-eb57a53a74aa.png)

Reply:
Yup, that's an error. I think there should only be one comma outside the parentheses.